Libs/DICOM/Core/ctkDICOMIndexer.h

```diff
@@ -107,7 +107,7 @@ public Q_SLOTS:
 private:
   Q_DECLARE_PRIVATE(ctkDICOMIndexer);
   Q_DISABLE_COPY(ctkDICOMIndexer);
-
+  int progressValue;
```

Comment:
Oh yeah that's absolutely makes sense. I somehow had in mind that it gets called for each file....